omap-tutorial/clientSimulator/scripts/README.md

```diff
@@ -0,0 +1,160 @@
+#ClientSimulator CRITERIA TESTS AUDIO TESTS FILES README
```

Comment:
I took 99% of this file's content and put it in the client simulator's README. Please refer to that file and add information relevant to your specific test suite.